### PR TITLE
Global Audio Controls

### PR DESCRIPTION
**Pull request by Raymond-Xia**
- Added volume slider to settings which controls the volume of the whole app
- Changed music/sfx sliders in pause menu to just one volume slider

image image

### CODE REVIEW

File: Assets/Scripts/Menu/VolumeSlider.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using UnityEngine.UI;
+
+public class VolumeSlider : MonoBehaviour
+{
+    private AudioController controller;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        controller = GameObject.Find("AudioController").GetComponent<AudioController>();
```
Comment (danielkim10) [inline Assets/Scripts/Menu/VolumeSlider.cs:13]: Ah ha!

**Review by danielkim10 — APPROVED** [review]
Would it be possible to save the slider value as a player pref? Or do you think this method is just simpler.

File: Assets/Scripts/Menu/VolumeSlider.cs (hunk 1 above)
Comment (Raymond-Xia) [reply Assets/Scripts/Menu/VolumeSlider.cs:13]: F*ckkk!!!!!!!!

**Comment by Raymond-Xia** [comment]
> Would it be possible to save the slider value as a player pref? Or do you think this method is just simpler.

I realized with most apps/games the volume setting is typically set to a default value on launch so it will just make more sense and also be easier.

**Pull request merged**